app/controllers/questions_controller.rb

```diff
@@ -28,10 +28,15 @@ def index
   end
 
   def new
-    if params[:legacy]
-      render 'editor/question'
-    else 
-      render 'editor/questionRich'
+    if current_user == nil
+      redirect_to '/login'
```

Comment:
Hi! Can  you add `/login?return_to/questions/new` so you get sent back after logging in? Thanks!!!